actions.md

```diff
@@ -131,9 +131,9 @@ These are the primary actions that a user of libaktualizr can perform through th
     - [x] Install an update on the primary
       - [ ] Install an OSTree update on the primary
       - [ ] Notify "reboot needed" after an OSTree update trigger
-      - [ ] Set new version to pending status after an OSTree update trigger
-      - [ ] Send EcuInstallationAppliedReport to server after an OSTree update trigger
-      - [ ] Uptane check for updates and manifest sends are disabled while an installation is pending reboot
+      - [x] Set new version to pending status after an OSTree update trigger
+      - [x] Send EcuInstallationAppliedReport to server after an OSTree update trigger
+      - [x] Uptane check for updates and manifest sends are disabled while an installation is pending reboot
```

Comment:
Would you mind listing in parentheses the file in which the test that covers these cases can be found, and also copying the same text here as a comment before the test? Minor, but for now that's the best I can do to link things together easily.

Reply:
Is it better now?

Reply:
Yes, but I have two further suggestions. First, the ticket number probably isn't necessary for now; I've only included ones that were requirements tickets. I don't know what the status of that system is, but I haven't seen any movement on that for a while. Second, could you add the text of the action to the code? As in, specifically put the items like "Set new version to pending status after an OSTree update trigger" in a comment before the test. It's not a perfect solution, but I just want to make sure that when we say the test is covered in some file, we can find exactly what we were thinking of when we made that claim.

Reply:
Updated again :)